ios/MusicControlManager.m

```diff
@@ -9,21 +9,56 @@
 @interface MusicControlManager ()
 
 @property (nonatomic, copy) NSString *artworkUrl;
+@property (nonatomic, strong) NSDictionary *mediaKeys;
 
 @end
 
+NSString * const MEDIA_PLAYBACK_RATE = @"playbackRate";
+
 @implementation MusicControlManager
 
 @synthesize bridge = _bridge;
 
 RCT_EXPORT_MODULE()
 
+- (NSDictionary *) mediaKeys {
+    if (_mediaKeys == nil) {
+        _mediaKeys = @{
+                @"album": MPMediaItemPropertyAlbumTitle,
+                @"trackCount": MPMediaItemPropertyAlbumTrackCount,
+                @"trackNumber": MPMediaItemPropertyAlbumTrackNumber,
+                @"artist": MPMediaItemPropertyArtist,
+                @"composer": MPMediaItemPropertyComposer,
+                @"discCount": MPMediaItemPropertyDiscCount,
+                @"discNumber": MPMediaItemPropertyDiscNumber,
+                @"genre": MPMediaItemPropertyGenre,
+                @"persistentID": MPMediaItemPropertyPersistentID,
+                @"duration": MPMediaItemPropertyPlaybackDuration,
+                @"title": MPMediaItemPropertyTitle,
+                @"elapsedPlaybackTime": MPNowPlayingInfoPropertyElapsedPlaybackTime,
+                @"playbackRate": MPNowPlayingInfoPropertyPlaybackRate,
```

Comment:
There is no elapsed time to store. It starts counting up from the time you give it. When you pause you have to tell it at what time to pause. That time comes from else where - the state of your playback outside the displayed info.

It's not about storing the information. I'm asking about the precedence. If I supply { state: STATE_PLAYING, speed: 0 } which one wins?